config/yaml-template/yurt-controller-manager.yaml

```diff
@@ -104,7 +104,9 @@ spec:
       serviceAccountName: __project_prefix__-controller-manager
       hostNetwork: true
       tolerations:
-      - operator: "Exists"
+      - key: "node-role.kubernetes.io/master"
+        effect: ""
+        operator: "Exists"
```

Comment:
Please comment the reason of adding `key: "node-role.kubernetes.io/master"`?

Reply:
same as previous one